lib/active_storage_validations/aspect_ratio_validator.rb

```diff
@@ -73,18 +73,18 @@ def is_valid?(record, attribute, metadata)
         return true if metadata[:width] > metadata[:height]
         add_error(record, attribute, :aspect_ratio_not_landscape)
 
-      else
-        if options[:with] =~ /is\_(\d*)\_(\d*)/
-          x = $1.to_i
-          y = $2.to_i
+      when /is_(\d*)_(\d*)/
+        x = $1.to_i
+        y = $2.to_i
 
-          return true if (x.to_f / y).round(PRECISION) == (metadata[:width].to_f / metadata[:height]).round(PRECISION)
+        return true if (x.to_f / y).round(PRECISION) == (metadata[:width].to_f / metadata[:height]).round(PRECISION)

```

Comment:
If `y` will be supplied as 0 this line will cause a devision by zero. What do you think about adding that as condition?
```ruby
return true if !y.zero? && (x.to_f / y).round(PRECISION) == (metadata[:width].to_f / metadata[:height]).round(PRECISION)
```
The code will then default into the invalid aspect ratio error.

Reply:
Done :) even if this case should not occur with the new regex